sentinel-adapter/sentinel-spring-webmvc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/config/SentinelWebMvcConfig.java

```diff
@@ -34,6 +34,11 @@
      */
     private boolean httpMethodSpecify;
 
+    /**
+     * Specify whether unify web context(i.e. use the default context name), and is true by default.
+     */
```

Comment:
LGTM, so we could add `@since 1.7.2` here :)

Reply:
Got, it added! and improve comment in demo.